docs/reference_guides/model_libraries/generic/costing/process_costing.rst

```diff
@@ -0,0 +1,891 @@
+Process Costing Library
```

Comment:
I think the name for this file (both the heading and file name) needs to be specific. E.g. "Process Costing using 'Process and Product Design Principles'" for the heading.

We hope(?) to have more libraries in future, so we need to name things with that in mind.